tests/v2/test_wakunode.nim

```diff
@@ -27,9 +27,13 @@ when defined(rln):
   import ../../waku/v2/protocol/waku_rln_relay/[waku_rln_relay_utils, waku_rln_relay_types]
 
 const RLNRELAY_PUBSUB_TOPIC = "waku/2/rlnrelay/proto"
+template sourceDir: string = currentSourcePath.parentDir()
+const KEY_PATH = sourceDir / "../../waku/v2/node/key.txt"
```

Comment:
Probably don't want to store the key/cert (a) in node code folder (b) as txt.

Are these just for test?

Reply:
yes